code/modules/mob/living/carbon/superior_animal/roach/types/hunter.dm

```diff
@@ -4,12 +4,15 @@
 	icon_state = "jager"
 
 	turns_per_move = 3
-	maxHealth = 25
-	health = 25
+	maxHealth = 35
+	health = 35
 	move_to_delay = 3.5
 
+	blattedin_revives_left = 2
+	waring_faction_multy = 2 //We are the figher!
+
 	melee_damage_lower = 3
-	melee_damage_upper = 10
+	melee_damage_upper = 8
```

Comment:
If they are meant to deal damage and be fighters, why lower their maximum damage and up their health pool ?

Reply:
to help chitmants spiders